polkadot/roadmap/implementers-guide/src/protocol-validator-disabling.md

```diff
@@ -0,0 +1,282 @@
+# Validator Disabling
+
+## Background
+
+As established in the [approval process](protocol-approval.md) dealing with bad parablocks is a three step process:
+
+1. Detection
+1. Escalation
+1. Consequences
+
+The main system responsible for dispensing **consequences** for malicious actors is the [dispute system](protocol-disputes.md) which eventually dispenses slash events. It is important to note the **high latency** of the punishment as it is only effective after 27 eras (27 days in Polkadot). Dispute concluding by itself does not immediately remove the validator from the active validator set.
+
+> **Note:** \
+> There was an additional mechanism of automatically chilling the validator which removed their intent to participate in the next election, but the removed validator could simply re-register his intent to validate.
```

Comment:
Not sure if relevant, but the nominations of the re-registered validator were also kept. This helps UI but may hinder nominators that are not fast to react (which is the case for most of the current nominators, it seems).

Reply:
I don't mention anywhere that nominators were unsubscribed so I assume that this bit is implied. Main point is they could re-register and just continue validating and think this point still comes across. So true but not as relevant.